Set/Models/GameModel.swift

```diff
@@ -122,15 +126,37 @@ struct GameModel {
         }
     }
     
-    mutating private func getBonus(for player: Player) {
+    private mutating func isFasterThanPreviousMatch() -> Bool {
```

Comment:
But why it's function in case it doesn't accept any values and can be computed property?